<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shadows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>911</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>31</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select a video file and click on  'Test'</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="trainsvm">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced Tools</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Mono L</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Suspicious Human Activity Detection System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Detecting Suspicious Activity:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="extract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="progressbox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Processing Video:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="status">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>911</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress Window:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="batchButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Batch Processing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
